src/GitElephant/Repository.php

```diff
@@ -1035,18 +1035,18 @@ public function pull($from = null, $ref = null, $rebase = true)
     }
 
     /**
-     * Fetch from and merge with another repository or a local branch
+     * Push changes to remote repository
      *
-     * @param string $to
-     * @param string $ref
+     * @param string $origin
+     * @param string $branch
      * @throws \RuntimeException
      * @throws \Symfony\Component\Process\Exception\LogicException
      * @throws \Symfony\Component\Process\Exception\InvalidArgumentException
      * @throws \Symfony\Component\Process\Exception\RuntimeException
      */
-    public function push($to = null, $ref = null)
+    public function push($origin = null, $branch = null)
```

Comment:
I'd like to keep $ref as parameter name, for consistency across the source code, and because this is not always a branch. You could push on a tag, on HEAD etc. So, to keep it close to git and its documentation I prefer $ref.

Thanks for your patience... :+1: